website/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start --host 0.0.0.0",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids"
+  },
+  "dependencies": {
+    "@docusaurus/core": "2.4.0",
```

Comment:
Wonder why we pin these exact dependencies? BTW there's version 2.4.1 now

Reply:
what do you mean? we pin everything.

I guess they could move to `devDependencies` since this isn't a library

Reply:
updated to v2.4.1